### PR TITLE
Update wayne_cow schema for #380

### PR DESCRIPTION
**Pull request by mkrump**
- update schema
- replace static description
- scrape documents

### CODE REVIEW

File: city_scrapers/spiders/wayne_cow.py [hunk 1]
```diff
@@ -1,12 +1,17 @@
...
 from datetime import datetime
+from urllib.parse import urljoin
+
 from city_scrapers.spider import Spider
 
 
 class Wayne_cowSpider(Spider):
     name = 'wayne_cow'
+    # TODO add agency_id
+    agency_id = '????'
```
Comment (mkrump) [inline city_scrapers/spiders/wayne_cow.py]: Wasn't sure what `agency_id` should be. Should be ready once that is updated.

Comment (r-wei) [reply city_scrapers/spiders/wayne_cow.py]: @diaholliday , what should be the official name for this agency? https://www.waynecounty.com/elected/commission/committee-of-the-whole.aspx

Comment (r-wei) [reply city_scrapers/spiders/wayne_cow.py]: @novellac , do you know the official name of this agency? https://www.waynecounty.com/elected/commission/committee-of-the-whole.aspx

Comment (diaholliday) [reply city_scrapers/spiders/wayne_cow.py]: Sorry, @r-wei, didn't realize that was your question (and was generally tied up in Detroit). The name is the Committee of the Whole (AKA COW). We could call it committee_whole unless it needs to be one word.

**Pull request merged**